src/Templating/TemplateRegistry.php

```diff
@@ -71,7 +71,7 @@ public function hasTemplate(string $name): bool
         return isset($this->templates[$name]);
     }
 
-    public function getTemplate($name): string
+    public function getTemplate(string $name): string
```

Comment:
Adding this return type is backwards-compatible, please contribute that to 3.x

Reply:
Is it ?

In 3.x, passing an int wouldnt end with an error.

But if we add the typehint, it will throw a typeerror if the developer has "strict type".

Reply:
> Adding this **return** type is backwards-compatible, please contribute that to 3.x

Reply:
Oh ok